README.md

```diff
@@ -91,16 +91,24 @@ Pwwka::Transmitter.send_message!(payload, routing_key)
 
 The payload should be a simple hash containing primitives. Don't send objects because the payload will be converted to JSON for sending.
 
-If an exception is raised sending your message, the bang form—`send_message!`—will pass that exception along to you and also blow up.  If
-you don't want that:
+Error handling can be configured by passing one of:
+
+```text
+  on_error: :raise (default)
+  on_error: :ignore
+  on_error: :resque
+```
+
+to `Pwwka::Transmitter.send_message!` (see "Error Handling", below). For example:
 
 ```ruby
-Pwwka::Transmitter.send_message_safely(payload, routing_key)
+Pwwka::Transmitter.send_message!(payload, routing_key, on_error: :ignore)
 ```
 
+
 #### Error Handling
 
-This method accepts several strategies for handling errors, pass in using the `on_error` parameter:
+This method accepts several strategies for handling errors, passed in using the `on_error` parameter:
 
   * `:raise`: Log the error and raise the exception received from Bunny. (default strategy)
```

Comment:
Do we need this still if we are listing them out above?  Or can we omit the list above?